kubernetes/helmfile.yaml

```diff
@@ -10,6 +10,8 @@ repositories:
     url: https://helm.elastic.co
   - name: bitnami
     url: https://charts.bitnami.com/bitnami
+  - name: tempo
+    url: https://github.com/grafana/helm-charts/
```

Comment:
we could also point to the correct helm chart directly and not to the whole grafana home page.

Reply:
Ref. https://github.com/dduportal/jenkins-examples/pull/3/files#r977880919